### PR TITLE
[Task]: Delete Help page

### PR DESCRIPTION
**Pull request by KobikahChandran**
## What this PR accomplishes
- Deletes Help page
- Deletes all references to the Help page from the About page

## Issue(s) addressed
- DATA-1341

## What needs review
- Help page is removed completely including from navigation
- All references to the Help page are removed
- Help page goes to 404 page and not 500

### CODE REVIEW

**Review by Alexandra-Green — CHANGES_REQUESTED** [review]
Can you remove the Colby help page and it's translations from the .po & .pot files as well?

File: ckanext/ontario_theme/plugin.py [hunk 1]
```diff
@@ -181,12 +181,6 @@ def resource_display_name(resource_dict):
 
 ckan.lib.helpers.resource_display_name = resource_display_name
 
-def help():
-    '''New help page for site.
-    '''
-    return render_template('home/help.html')
-
-
 def csv_dump():
```
Comment (Alexandra-Green) [inline ckanext/ontario_theme/plugin.py:184]: Can you add an additional new line above line 184? So there are 2 new lines before `def csv_dump():`

**Comment by Alexandra-Green** [comment]
@aimalkhan do we need to update the version # in the `setup.py`?

**Comment by aimalkhan** [comment]
> @aimalkhan do we need to update the version # in the `setup.py`?

@KobikahChandran update it to 2.1.32 please.

**Review by Alexandra-Green — APPROVED** [review]

**Pull request merged**